editoast/src/tests/example_rolling_stock_1.json

```diff
@@ -980,5 +980,7 @@
         "unit": ""
     },
     "energy_sources": [],
-    "power_restrictions": {}
+    "power_restrictions": {},
+    "electrical_power_startup_time": 5.0,
+    "raise_pantograph_time": 15.0
```

Comment:
The unit tests in views/rolling_stock.rs do depend on those .json, and are automatically verifying the whole object almost everywhere, so you are indeed updating and fixing those unit tests by adding those fields :)
However, since those fields seem to be nullable, you seem to be missing the test case where they are null. Is it possible to add this test case so as to showcase that it indeed does still work ?
I would also encourage changing the fields' values to have different values in the 3 .json. Indeed, in the update test, we take one rolling stock json and update it with the values of another. Putting different values would check that those fields are indeed updated.

Reply:
Agreed on the idea. But rather than creating some other JSON files for that specific case, I'd be more in favor of loading this JSON and changing the key value in the test case (mutating the `serde_json::Value` or the rolling stock directly) so that:
* we avoid duplicating files that we need to maintain later
* it highlights the case that's being tested while reading the code. 
Wdyt?

Reply:
Was thinking of that as well for the null test case, lgtm for sure :)